datafusion/proto/proto/datafusion.proto

```diff
@@ -798,6 +799,19 @@ message UnnestExecNode {
   UnnestOptions options = 5;
 }
 
+message MemoryExecNode {
```

Comment:
@jayzhan211 am i going in right direction ?

Thanks,

Reply:
I miss this message, probably because it is in draft mode

Reply:
It seems so, this should be similar to other exec node